### PR TITLE
Workaround fix for issue #2441

### PR DESCRIPTION
**Pull request by goJoWe16**
These changes fix issue #2441 in a workaround style.

There still exist the problem of issue #2408 --> When this problem is fixed, this workaround may not be needed anymore.

### CODE REVIEW

File: doc/news/_preparation_next_release.md [hunk 1]
```diff
@@ -211,6 +211,7 @@ you up to date with the multi-language support provided by Elektra.
 ## Tools
 
 - `kdb get -v` now displays if the resulting value is a default-value defined by the metadata of the key. _(Thomas Bretterbauer)_
+- `ẁeb-ui` issue #2441 is fixed in from of a workaround
```
Comment (sanssecours) [inline doc/news/_preparation_next_release.md]: Looks like a recent change to the `master` branch introduced a merge conflict. Please 

1. rebase on `master`,
2. fix the spelling mistake (`from` → `form`), and
3. add your name at the end of this news item

.

Comment (goJoWe16) [reply doc/news/_preparation_next_release.md]: I have made these changes and pushed them

**Comment by markus2330** [comment]
Thank you for the fix!

**Comment by markus2330** [comment]
Thank you for the fix!

**Pull request merged**